packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -60,6 +68,7 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
     ExpectedColumns.PUBLICATION_DATE,
     queryResultBundle,
   )
+  const doiColIndex = getFieldIndex(ExpectedColumns.DOI, queryResultBundle)
 
   const PublicationCard = ({ pub }: PublicationCardProps) => (
```

Comment:
I just noticed that this component is declared inside of another component. This violates the [no-unstable-nested-component](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md) rule. Can you move the `PublicationCard` definition outside of the `RecentPublicationsGrid` function component?

Also I recommend ensuring you have the ESLint VSCode extension enabled to highlight these issues for you.

Reply:
This has come up a few times from multiple engineers, so I created PORTALS-3369 to change the lint warning to an error here, to make it easier to catch.